### ISSUE TITLE
Layout/EmptyLinesAroundClassBody false positive

### ISSUE THREAD

**Issue by mockdeep** [bbatsov/rubocop#5659]
I've got a class with a long name inheriting from a deeply nested class, so I'm wrapping to the next line:

```ruby
class ValidateExistenceOfMatcher <
  Shoulda::Matchers::ActiveModel::ValidationMatcher

  def matches?(subject)
    super
    disallows_value_of(nil)
  end

  def simple_description
    "validate that :#{@attribute} is not nil"
  end

end
```

--------

## Expected behavior

I would expect rubocop not to report a violation of the `Layout/EmptyLinesAroundClassBody` rule.

## Actual behavior

It reports: `Layout/EmptyLinesAroundClassBody: Empty line missing at class body beginning.`

## Steps to reproduce the problem

Run rubocop on the above code with the above code and the following config:

```yml
Layout/EmptyLinesAroundClassBody: { EnforcedStyle: empty_lines }
```

## RuboCop version

```
$ rubocop -V
0.52.1 (using Parser 2.5.0.2, running on ruby 2.4.3 x86_64-linux)
```

**Comment by AlexWayfer** [comment]
It seems like there is regression with original example and:

```
0.67.2 (using Parser 2.6.2.1, running on ruby 2.6.2 x86_64-linux)
```
